grafana-plugin/package.json

```diff
@@ -27,7 +27,8 @@
     "ci-report": "grafana-toolkit plugin:ci-report",
     "start": "yarn watch",
     "plop": "plop",
-    "setversion": "setversion"
+    "setversion": "setversion",
+    "tsc": "tsc --noEmit"
```

Comment:
FYI in my PR it's named `typecheck`, I don't know what is better...=)

Reply:
Doesn't matter, let's go with `typecheck` :)